integrasjon/rest-klient/src/main/java/no/nav/vedtak/felles/integrasjon/rest/FpApplication.java

```diff
@@ -24,7 +24,8 @@ public enum FpApplication {
 
     private static final Environment ENV = Environment.current();
     private static final Cluster CLUSTER  = ENV.getCluster();
-    private static final Namespace NAMESPACE = ENV.getNamespace();
+    // FpApplication brukes til å kalle apps i namespace foreldrepenger - ikke riktig å bruke ENV/namespace
+    private static final Namespace FORELDREPENGER = Namespace.of("teamforeldrepenger");
```

Comment:
Dette vil nok skape problemer med fptilbake som K9 starter i sitt eget namespace. Så da er man avhengig av ENV.getNamespace().

Reply:
Det vil ikke skape problemer, men heller løse det problemet vi har nå når det brukes current.
Når target for kall er en av FpApplication så skal de kjøre i namespace foreldrepenger